CRM/Contact/Form/Edit/CustomData.php

```diff
@@ -37,7 +36,7 @@ public static function preProcess(&$form) {
       $form->assign("blockName", $form->_addBlockName);
     }
 
-    CRM_Custom_Form_CustomData::preProcess($form, NULL, $form->_subType, NULL,
+    CRM_Custom_Form_CustomData::preProcess($form, NULL, NULL, NULL,
```

Comment:
Why is this changed to NULL?

Reply:
because the `preProcess` function retrieves it from the url with `$form->_subType = CRM_Utils_Request::retrieve('subType', 'String');` if it is NULL

Reply:
see https://github.com/civicrm/civicrm-core/pull/26833/files#diff-931e65674ed680adf93d7ba4727419ec32b5fd4636ac84109dd9621f89e20e0dR91

Reply:
Ah ok.